deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_reader.erl

```diff
@@ -763,11 +766,11 @@ info_internal(frame_max, #v1{connection = #v1_connection{frame_max = Val}}) ->
 info_internal(timeout, #v1{connection = #v1_connection{timeout_sec = Val}}) ->
     Val;
 info_internal(user,
-              #v1{connection = #v1_connection{user = #user{username = none}}}) ->
-    '';
-info_internal(username,
-              #v1{connection = #v1_connection{user = #user{username = Val}}}) ->
+              #v1{connection = #v1_connection{user = {user, Val, _, _}}}) ->
```

Comment:
Why was this changed from using the `#user{}` record?

Reply:
There were two changes. One was to remove `info_internal(username, ...)` given that `username` was not in the list of info items. And with regards the second change, which is what you are referring to, you are totally right. It is far more appropriate to use the original record expression. I have changed the code. Thanks @lukebakken